charts/incubator/privatebin/values.yaml

```diff
@@ -0,0 +1,60 @@
+image:
+  repository: privatebin/pdo
+  pullPolicy: IfNotPresent
+  tag: 1.4.0@sha256:5fe4b94286dcadbd0e576418f619205463030754e257863ec430eb7eb3e957fd
+
+env:
+  # See also: https://github.com/PrivateBin/docker-nginx-fpm-alpine#timezone-settings
+  TZ: "{{ .Values.TZ }}"
+  PHP_TZ: "{{ .Values.TZ }}"
+
+securityContext:
+  # This image supports running in read-only mode. Using this reduces the attack
+  # surface slightly, since an exploit in one of the images services can't
+  # overwrite arbitrary files in the container. Only /tmp, /var/tmp, /var/run &
+  # /srv/data may be written into.
+  # See also: https://github.com/PrivateBin/docker-nginx-fpm-alpine#running-the-image
+  readOnlyRootFilesystem: true
+  privileged: false
+  allowPrivilegeEscalation: false
```

Comment:
```suggestion
```
Those are the defaults so you can omit them

Reply:
You have to define podSecurityContext here, as I saw it uses different user/group based on questions.yaml